src/main/java/uk/gov/hmcts/darts/notification/NotificationConstants.java

```diff
@@ -10,7 +10,7 @@ public static class ParameterMapValues {
         public static final String COURTHOUSE = "courthouse";
         public static final String DEFENDANTS = "defendants";
         public static final String HEARING_DATE = "hearing_date";
-        public static final String AUDIO_START_TIME = "start_time";
+        public static String AUDIO_START_TIME = "start_time";
```

Comment:
why was final removed?

Reply:
It was being prototyped, I forgot to put it back. Will amend it now

Reply:
It was initially removed as part of a test scenario. It should have been put this back. Now corrected